modules/test_user_dashboard/app/models/test_user_dashboard/tud_account.rb

```diff
@@ -10,7 +10,7 @@ class TudAccount < ApplicationRecord
     validates :email, uniqueness: true
     validate :valid_id_types
 
-    serialize :services
+    serialize :services, coder: YAML
```

Comment:
YAML was [the default prior to 7.1](https://guides.rubyonrails.org/configuring.html#config-active-record-default-column-serializer), so I restored it here. However, maybe we should use something else? Can we use the same coder as everywhere else (`JsonMarshal::Marshaller`)?